File: BackEnd/ajax/Edificio.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Edificio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'nombre': null,
    'idCentroMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Edificio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPisos',
    'idCentroMedico': null,
    'idEdificio': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Edificio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizarPiso',
    'descripcion': null,
    'idPiso': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Edificio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarPiso',
    'descripcion': null,
    'idEdificio': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Edificio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPiso',
    'idPiso': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Edificio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentro',
    'idCentroMedico': null,
    'nombreCentro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Edificio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listar',
    'idEdificio': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

File: BackEnd/postman/Edificio.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Edificio'
data={
  'accion':'crear',
  'nombre': None,
  'idCentroMedico': None,
}
print postear(data,url)
print

url='Edificio'
data={
  'accion':'actualizarPiso',
  'descripcion': None,
  'idPiso': None,
}
print postear(data,url)
print

url='Edificio'
data={
  'accion':'agregarPiso',
  'descripcion': None,
  'idEdificio': None,
}
print postear(data,url)
print

url='Edificio'
data={
  'accion':'listarPiso',
  'idPiso': None,
}
print postear(data,url)
print

url='Edificio'
data={
  'accion':'listarPorCentro',
  'idCentroMedico': None,
  'nombreCentro': None,
}
print postear(data,url)
print

url='Edificio'
data={
  'accion':'listar',
  'idEdificio': None,
}
print postear(data,url)
print
